### PR TITLE
MDN feature pages for SVGAnimatedTransformList

### PR DESCRIPTION
**Pull request by yashrajbharti**
part of [Missing baseline pages](https://github.com/openwebdocs/project/issues/214) project

### CODE REVIEW

File: files/en-us/web/api/svganimatedtransformlist/baseval/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+slug: Web/API/SVGAnimatedTransformList/baseVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedTransformList.baseVal
+---
+
+{{APIRef("SVG")}}
+
+The **`SVGAnimatedTransformList.baseVal`** read-only property of the {{domxref("SVGAnimatedTransformList")}} interface represents the non-animated value of the `transform` attribute of an SVG element.
+
+This property provides access to the static value of the `transform` attribute, as set by the `transform` or similar presentation attributes (such as `gradientTransform` or `patternTransform`) of an SVG element.
```
Comment (estelle) [inline files/en-us/web/api/svganimatedtransformlist/baseval/index.md:13]:
```suggestion
This property reflects the SVG element's {{SVGAttr("transform")}}, the {{SVGElement("linearGradient")}} or {{SVGElement("radialGradient")}} element's {{SVGAttr("gradientTransform")}} attribute, or the {{SVGElement("pattern")}} element's {{SVGAttr("patternTransform")}} attribute value as a readonly {{ domxref("SVGTransformList") }}, providing access to a static {{ domxref("SVGTransform") }} for each transform function set on the SVG element.
```

File: files/en-us/web/api/svganimatedtransformlist/baseval/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```js
+// Get the rectangle element
+const rect = document.getElementById("myRect");
+
+// Access the non-animated transform list of the element
+const baseTransforms = rect.transform.baseVal;
+
+// Log the base transforms to the console
+console.log(baseTransforms);
```
Comment (estelle) [inline files/en-us/web/api/svganimatedtransformlist/baseval/index.md:29]:
```suggestion
console.dir(baseTransforms);
```
for strings, use `log`. If an objects or object-like, use `dir`.

File: files/en-us/web/api/svganimatedtransformlist/baseval/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/api/svganimatedtransformlist/baseval/index.md:13]: similar edit for anim

**Review by estelle — COMMENTED** [review]
looks pretty good. 

I made a suggestion on the base for adding that it reflects attributes, which attributes are reflected on which elements, and a bit more on what is returned.

**Comment by yashrajbharti** [comment]
Thanks a lot, I will make sure to follow them, and fix this for all the SVGAnimated PRs. I aim to write with clarity and this helps a lot!

**Review by estelle — APPROVED** [review]
Thank you! 🎉

**Pull request merged**